Modules/MUON/MID/src/CalibQcTask.cxx

```diff
@@ -214,18 +214,20 @@ void CalibQcTask::initialize(o2::framework::InitContext& /*ctx*/)
 void CalibQcTask::startOfActivity(Activity& /*activity*/)
 {
   ILOG(Info, Support) << "startOfActivity" << ENDM;
+  std::cout << "!!!! START startOfActivity a CalibQcTask !!!! " << std::endl;
 }
 
 void CalibQcTask::startOfCycle()
 {
   ILOG(Info, Support) << "startOfCycle" << ENDM;
+  std::cout << "!!!! START startOfCycle CalibQcTask !!!! " << std::endl;
 }
 
 void CalibQcTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
-  // std::cout << "!!!! START monitorData CalibQcTask !!!! " << std::endl;
+  std::cout << "!!!! START monitorData CalibQcTask !!!! " << std::endl;
```

Comment:
logging and printing should not happen in `monitorData` (apart for the development). It results in a very large amount of logs.